docs/compendium/substitute.md

```diff
@@ -1,19 +1,30 @@
 # Substitute compendia
 
-introduction...
+Introduction to substitution of two compendia. <br/>
+A new compendium will be the result. Merged out of choosen data and files of two existing compendia, the new compendium will give the user an opportunity to exchange datasets or analysis and more or less create a new compendium.
```

Comment:
"more or less" - remove phrase. This is technical documentation, no room for uncertainty.
Also "create a new compendium." is a repetition of the first sentence in this paragraph.